crazy_miner/telemedicine_views.py

```
@@ -0,0 +1,386 @@
+
+    return render(request, 'telemedicine/verification_order.html', context)
+
+
+class download_order_file(APIView):
+    permission_classes = [permissions.AllowAny]
+
+    def get(self, request, national_code):
+        order = get_object_or_404(Order, national_code=national_code)
+        if order:
+            file_path = os.path.join(settings.MEDIA_ROOT, 'pdf', 'order', f'order_{national_code}.pdf')
+            response = FileResponse(open(file_path, 'rb'), as_attachment=True, filename=f'order_{national_code}.pdf')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)